src/main/java/uk/gov/hmcts/divorce/document/content/GenerateJudicialSeparationCORefusedForAmendmentCoverLetter.java

```diff
@@ -55,9 +66,13 @@ public void generateAndUpdateCaseData(final CaseData caseData,
 
         log.info("Generating Judicial Separation Conditional Order Refused Cover Letter for case id {} ", caseId);
 
+        final DocumentType coverLetter = caseData.getApplication().isPaperCase() && applicant.isRepresented()
```

Comment:
Is the isPaperCase() check required here? JS cases are always paper / offline

Reply:
Resolved